core/src/sigverify_stage.rs

```diff
@@ -248,20 +248,27 @@ impl SigVerifyStage {
             num_packets,
         );
 
-        let mut dedup_time = Measure::start("sigverify_dedup_time");
-        let dedup_fail = deduper.dedup_packets(&mut batches) as usize;
-        dedup_time.stop();
-        let num_unique = num_packets.saturating_sub(dedup_fail);
+        use rand::{thread_rng, Rng};
+        const MAX_BATCHES: usize = 2000;
```

Comment:
Last time I checked times for dedup and verify on the cluster it was roughly 300ns/packet for dedup and 25us/packet for verify. So maybe 
```
MAX_DEDUP_BATCHES = 50ms/(300ns/packet) = 166666 packets ~ 1300 batches?
MAX_SIGVERIFY_PACKETS = 50ms/(25us/packet) = 2000 packets?
```
to keep each sigverify iteration below 100ms?

As a next iteration after this PR we could use a `DynamicPacketToProcessThreshold` to target a particular runtime for dedup and verify? (though I'm worried that it'll update to slowly - there's a phase change that happens when getting close to a leader slot)

Reply:
ok, used those constants.